addons/compute_worker/gpu_uniforms/gpu_uniform.gd

```diff
@@ -195,9 +240,17 @@ func byte_array_to_vec3_array(bytes: PackedByteArray) -> PackedVector3Array:
 func float_array_to_byte_array_64(array: Array[float]) -> PackedByteArray:
 	var bytes = PackedFloat64Array(array).to_byte_array()
 	return bytes
-	
+
+## Convert an array of Floats to GLSL equivalent `double[]`
```

Comment:
the doc comment here reflects something different than what the function does

Reply:
Yeah I had to ditch pretty fast today, will be fixed TMRW as well

Reply:
All good. Thanks for taking the time to add this stuff in!

Reply:
It was fun! It's still not perfect but I think it works? Had some alignment issues and general confusion because I know nothing about glsl bit alignments, so worked of your comments and some deduction work!

Reply:
That's awesome! I haven't tested it yet, just reviewed the code. If you want a reference for uniform buffer alignments, you can view the OpenGL docs about it [here](https://registry.khronos.org/OpenGL/specs/gl/glspec45.core.pdf#page=158), which continues for the next few pages there.